packages/angular_devkit/benchmark/src/main_spec.ts

```diff
@@ -0,0 +1,144 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { existsSync, readFileSync, unlinkSync, writeFileSync } from 'fs';
+import { basename, dirname, join } from 'path';
+import { main } from './main';
+// tslint:disable-next-line:no-implicit-dependencies
```

Comment:
strange I thought this was `no-require-imports`

Reply:
Disabling TSLint here was because the `temp` dependency is not declared in the dependencies or devdependencies of this package. Rather, it's a top level dependency of the monorepo that we use for tests, and in other tests where it is used (`/core/node/host_spec.ts`) this is how we import it. Maybe we should make these explicit in the devDependencies?